client/setec/store.go

```diff
@@ -513,6 +462,17 @@ func StaticFile(path string) (Secret, error) {
 	return func() []byte { return bs }, nil
 }
 
+func panicOnUpdate[T any]([]byte) (T, error) { panic("unexpected value update") }
+
+// StaticUpdater returns an [Updater] that vends the specified fixed value.
+// The value reported by the updater never changes.
+func StaticUpdater[T any](fixedValue T) *Updater[T] {
+	return &Updater[T]{
+		newValue: panicOnUpdate[T],
```

Comment:
To be sure I understand: barring a bug in this library, this should never get called, correct? In particular, there's no way for a user of this library to hold things wrong enough to trigger the panic?

Reply:
That is correct—the only way that could happen is if a new secret value were delivered, which cannot happen as there is no Store. This is mainly here as documentation.